commands/search-slapdash-platform/command.toml

```diff
@@ -0,0 +1,15 @@
+name = "Search Slapdash Platform Docs"
+description = "Command to search through Slapdash Platform documentation and open them in your browser."
+endpointURL = "https://slapcommands.vercel.app/api/docsearch/slapdash-platform"
```

Comment:
1. Would be nice to change the sorting for options. For example, it will probably make sense to first show the top-level pages, then 2nd-level, 3-rd level, and so on. 

![CleanShot 2022-03-28 at 13 18 49](https://user-images.githubusercontent.com/57078134/160396283-70e60610-fcf4-426d-b6d5-3e7797bb8d62.png)

2. Notice these options take me to 404 pages. I suspect something is off with your storage. These should likely take me to the individual pages at https://platform.slapdash.com/command-tutorials.

![CleanShot 2022-03-28 at 13 22 11](https://user-images.githubusercontent.com/57078134/160396829-bf3ca27a-a6f7-42b3-af5e-0a9118610aa3.png)


3. What it be possible to rename Root to Home? Also, I think showing `subtitle` in this case is imprecise. Would be good to just name it "Home" and remove the subtitle. 
![CleanShot 2022-03-28 at 13 16 54](https://user-images.githubusercontent.com/57078134/160396036-6ef8300b-d516-48ae-bfba-a312db2bd6fd.png)

Reply:
Ok so, this data is generated by a script that basically walks through the slapdash docs GitHub repository ([source code](https://github.com/AnishDe12020/slapcommands/blob/main/src/scraper/docs/github.ts)). 

The only possible way I see right now is to manually go through the json now and remove the generation script for Slapdash (and add to it as the docs get updated). This can be quite some work though and I can't assure up to date docs

PS: I am a little busy now, will try to do these ASAP but it might be the first week of next month

Reply:
I have updated the scraper script and the data, these issues should be fixed now

Reply:
Thanks, @AnishDe12020!

1. What do you think about my suggestion earlier to sort the options by depth level? 
2. I still get a 404 when I try to go to one of the tutorials. This is the URL it takes me to https://platform.slapdash.com/commands-tutorials/copy-special-character. I believe it should be "command-tutorials" (without "s").

![CleanShot 2022-04-05 at 10 33 21](https://user-images.githubusercontent.com/57078134/161724757-d1373bdf-ffda-418d-930a-7980b1c2db0d.png)

Reply:
I have fixed the 404.

I did find a problem with the docs repo though. It seems like all the files in the `build-commands` directory are duplicates. This is also causing some duplicates of the data I am generating. For the time being, I have deleted this folder locally and generated the data.

Regarding sorting data according to depth, I think that is an awesome idea! I am working on it and I will give you the updates

Reply:
@taras-slapdash I have added the sorting part

Reply:
@AnishDe12020 I have now deleted the `build-commands` folder from docs. It was left there by mistake as part of some refactoring. 

Let us know if you find any other issues and we'll fix them.